src/components/tables/TableRowReducers.ts

```diff
@@ -18,13 +19,14 @@ export const tableRowReducer = (state: ITableRowState = tableRowInitialState, ac
       return {
         id: action.payload.id,
         tableId: action.payload.tableId,
+        rowId: action.payload.rowId,
         opened: false,
         selected: false,
       };
     case TableRowActions.select:
       if (state.tableId === action.payload.tableId) {
         return state.id === action.payload.id
-          ? { ...state, selected: true, opened: !!action.payload.isCollapsible && !state.opened }
+          ? _.extend({}, state, {selected: true, opened: !!action.payload.isCollapsible && !state.opened})
```

Comment:
it wasn't working or you forgot to change it back? the spread operator should work fine here, we should leave it there